### PR TITLE
Update spaceapi to 0.3

### LINKED ISSUE #40: Upgrade spaceapi dependency
0.2.0 is out :stuck_out_tongue:

### PR DESCRIPTION
**Pull request by rnestler**
Fixes #40

### CODE REVIEW

**Comment by dbrgn** [comment]
Good to merge!

**Comment by rnestler** [comment]
Should I bump the version in `Cargo.toml` to and update `CHANGELOG.md`? Since this is a breaking release we need to bump to 0.3, not 0.2.1

**Comment by dbrgn** [comment]
Ah, true, we expose `spaceapi` to users. :+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ Possible log types:
 - `[security]` to invite users to upgrade in case of vulnerabilities.
 
 
-### UNRELEASED
+### v0.3.0 UNRELEASED
```
Comment (rnestler) [inline CHANGELOG.md]: @dbrgn I changed it to 0.3 here, since it is clear now that we will release 0.3.

**Comment by dbrgn** [comment]
Merge :)

**Pull request merged**